### PR TITLE
[ECUX-308] Add regexes across all coins in the resolver keys JSON

### PR DESCRIPTION
**Pull request by udhriaznov**
## Background

We have to ensure all coin addresses have a valid regex in the following config file https://github.com/unstoppabledomains/uns/blob/main/resolver-keys.json

We can skip non-coin addresses, there should be around 88 coin addresses to update.

It is critical to include valid regexes to ensure we store valid values.

## Related Issue

https://linear.app/unstoppable-domains/issue/ECUX-308/add-regexes-across-all-coins-in-the-resolver-keys-json

### CODE REVIEW

**Review by iurevych — APPROVED** [review]
👍 LGTM but I'd also wait on @aquiladev confirmation.

File: resolver-keys.json [hunk 1]
```diff
@@ -1223,12 +1223,12 @@
...
     "crypto.CEL.version.MATIC.address": {
       "deprecatedKeyName": "CEL_MATIC",
-      "validationRegex": null,
+      "validationRegex": "^[a-zA-Z0-9]*$",
       "deprecated": false
     },
     "crypto.CEL.version.FANTOM.address": {
       "deprecatedKeyName": "CEL_FANTOM",
-      "validationRegex": null,
+      "validationRegex": "^[a-zA-Z0-9]*$",
```
Comment (iurevych) [inline resolver-keys.json:1231]: I think we should also add a character size limit here. ONG example: `"^[a-zA-Z0-9]{34}$"`

**Review by iurevych — APPROVED** [review]
👍

**Comment by aquiladev** [comment]
@iurevych LGTM 
@unstoppabledomains/developer-tools-team responsible for `resolver-keys.json`
@DancingAxolotl FV

**Pull request merged**